projects/gitpython/project.yaml

```diff
@@ -1,10 +1,12 @@
-fuzzing_engines:
-- libfuzzer
-homepage: https://github.com/gitpython-developers/gitpython
+homepage: "https://github.com/gitpython-developers/gitpython"
 language: python
-main_repo: https://github.com/gitpython-developers/gitpython
+primary_contact: "byronimo@gmail.com"
+auto_ccs:
+  - "david.js.lakin@gmail.com"
+  - "eliah.kagan@gmail.com"
+main_repo: "https://github.com/gitpython-developers/gitpython"
+fuzzing_engines:
+  - libfuzzer
 sanitizers:
-- address
-- undefined
-vendor_ccs:
-- david@adalogics.com
+  - address
+  - undefined
```

Comment:
Nit: does `gitpython` have any native code being compiled with sanitizers from OSS-Fuzz, or is it pure Python? If it's pure Python, then please remove `undefined` from the list here

Reply:
I believe it is pure Gitpython is pure python code, no native extensions. I'll update this shortly.

CC: @Byron @EliahKagan

Reply:
Yes, that's correct. GitPython is pure Python with no native extensions.

(Its dependencies [gitdb](https://github.com/gitpython-developers/gitdb) and [smmap](https://github.com/gitpython-developers/smmap) are likewise pure Python. The gitdb dependency has in the past been possible to attempt to make faster through the use of [gitdb-speedups](https://github.com/gitpython-developers/gitdb-speedups), which is a native code module. That project is archived and no longer developed or recommended, and it's not known if, or how well, that works anymore. Furthermore, it speeds up functionality of gitdb that is no longer widely used in GitPython and itself no longer recommended, since today the main significance of gitdb and smmap are helper types rather than the actual sliding-window memory manager and in-memory database.)

Reply:
Thanks, @EliahKagan! I'll update this PR to remove `undefined` from this list.

FWIW, `gitdb` is integrated with this project as well (see 
https://introspector.oss-fuzz.com/project-profile?project=gitdb) and might be worth upstreaming in a similar fashion, but we can talk about that elsewhere.

Reply:
Done 👍